iterations.r

```diff
@@ -40,30 +42,43 @@ stations_metadata_df <-
   stations_metadata %>% 
   transform_metadata_to_df(.)
 
+stations_metadata_df
 
 #### 3: Testing metadata
 source("functions/data_tests.r")
 test_stations_metadata(stations_metadata_df)
 
+### 4: test
+GQL(
```

Comment:
This does not work since you have not yet run the `source("gql-queries/vol_qry.r")` line. It would work if you added it after running that line however :)